### PR TITLE
CF10 CF11

### PR DESCRIPTION
**Pull request by sanaullah**
Error detail + message empty

### CODE REVIEW

File: tests/specs/integration/ModuleTests.cfc [hunk 1]
```diff
@@ -77,7 +77,7 @@ component extends="coldbox.system.testing.BaseTestCase" appMapping="/cbTestHarne
 
 			it( "should not load mdoules that have been excluded, even in bundles", function(){
 				var config = getController().getSetting( "modules" );
-				expect(	config ).notToHaveKey( 'excludedmod' )
+				expect(	config ).notToHaveKey( 'excludedmod' );
```
Comment (lmajano) [inline tests/specs/integration/ModuleTests.cfc]: this is good.

File: test-harness/handlers/testerror.cfc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+component output="false"{
```
Comment (lmajano) [inline test-harness/handlers/testerror.cfc]: What's the point of this handler?

Comment (sanaullah) [inline test-harness/handlers/testerror.cfc]: testing coldbox error template and Visually we can analise error info. 

At the moment in CF11 the error template does not display error-detail + error-message

Comment (lmajano) [inline test-harness/handlers/testerror.cfc]: AHH OK, you are adding an exception, so you can visualize stuff? Got it.

**Pull request merged**